pkg/clients/gitlab/git.go

```diff
@@ -166,3 +166,28 @@ func (gc *GitlabClient) CreateFile(pathToFile, fileContent, branchName string) (
 
 	return file, nil
 }
+
+func (gc *GitlabClient) CreateFileInProject(projectID, pathToFile, fileContent, branchName string) (*gitlab.FileInfo, error) {
```

Comment:
would it make sense to unify this method and `CreateFile` method above? it looks like a duplicated code and keeping this method would probably make more sense (because of the `projectID` parameter, that makes that method more dynamic and reusable across multiple tests)

Reply:
Definitely! That was an oversight. I see that the original `CreateFile` method is only used in one place so it should be a quick and easy change